api/api.proto

```diff
@@ -498,6 +498,8 @@ message ListGroupsRequest {
   string cursor = 2;
   // Max number of groups to return. Between 1 and 100.
   google.protobuf.Int32Value limit = 3;
+  // Whether to include full groups in the listing.
+  google.protobuf.BoolValue omit_full = 4;
```

Comment:
I think this can be a `bool` rather than `google.protobuf.BoolValue` since the absence of the value would be `false` and therefore the old/default behaviour of the server.

Reply:
Right, there's no need to use the protobuf wrapper, changed to vanilla `bool`